rasa/core/policies/ted_policy.py

```diff
@@ -328,6 +337,11 @@ def __init__(
 
         if not featurizer:
             featurizer = self._standard_featurizer(max_history)
+        else:
+            if max_history:
+                featurizer = self._carring_max_history_to_featurizer(
+                    featurizer.state_featurizer, max_history
+                )
```

Comment:
would it be simpler?
```suggestion
            if isinstance(featurizer, MaxHistoryTrackerFeaturizer) and max_history:
                featurizer.max_history = max.history
```

Reply:
Sinse this code is inside else statement, it means that a featurizer is instantiated as a TrackerFeaturizer that doesn't have a max_history parameter.

Reply:
if passed featurizer is not a `MaxHistoryTrackerFeaturizer`, you shouldn't add max_history

Reply:
So if i do just as is shown in the example made in issue description, just making the name as MaxHistoryTrackerFeaturizer it will be instantiated as one? If this is true, sorry, i must have passed over that while i was analyzing the code.

Reply:
I did that code based on the idea that in the code there was no place where a subclass of TrackerFeaturizer was instantiated by the user and in the comments of the issue someone say that they were using only "Genie's featurizer" for everything.

Reply:
> making the name as MaxHistoryTrackerFeaturizer it will be instantiated as one?
I don't remember, but should be. Requires checking

Reply:
I checked and it works with the name of the featurizer. Thanks for your support.